cmsimple/metaconfig.php

```diff
@@ -7,6 +7,7 @@
 $mcf['site']['compat']="hidden";
 $mcf['language']['default']="function:XH_availableLocalizations";
 $mcf['language']['2nd_lang_names']="hidden";
+$mcf['language']['menu']=XH_registeredLanguagemenuPlugins();
```

Comment:
This looks wrong. As is, the function `XH_registeredLanguagemenuPlugins()` will be called when metaconfig.php is included. Should this be:
```suggestion
$mcf['language']['menu']="function:XH_registeredLanguagemenuPlugins";
```

Reply:
Ok, although it works without problems